sherlock/resources/data.json

```diff
@@ -773,6 +773,13 @@
     "username_claimed": "adam",
     "username_unclaimed": "noonewouldeverusethis"
   },
+  "Imgur": {
+    "errorType": "No results",
+    "url": "https://www.imgur.com/user/{}",
+    "urlMain": "https://www.imgur.com",
+    "username_claimed": "blue",
+    "username_unclaimed": "noonewouldeverusethis",
```

Comment:
Remove the comma at the end of this line.

Reply:
Ohh ok doing it when I'm home